control_panel_api/tools.py

```diff
@@ -0,0 +1,66 @@
+import secrets
+
+from django.conf import settings
+from rest_framework.exceptions import APIException
+
+from control_panel_api.helm import Helm
+
+
+SUPPORTED_TOOL_NAMES = [
+    'rstudio',
+]
+
+
+class UnsupportedToolException(APIException):
+    tools_list = ", ".join(SUPPORTED_TOOL_NAMES)
+
+    status_code = 400
+    default_detail = f'Unsupported tool, tool_name must be one of: {tools_list}'
+    default_code = 'unsupported_tool'
+
+
+class Tool():
```

Comment:
Although all classes in Python 3 are new-style classes, you should still use `class Tool(object):` - at least until Python 2 has gone away

Reply:
Oh so this is for retro-compatibility reason. Copy that 👍